get_instagram.py

```diff
@@ -0,0 +1,21 @@
+from InstagramAPI import InstagramAPI
+from key import *
+
+
+def user():
+	api = InstagramAPI(username=user_name, password=password)
+	api.login()
+	try:
+		user_id = api.username_id
+		followers = api.getTotalFollowers(user_id)
```

Comment:
Please create a separate function which returns a list of followers and followings.

Reply:
created